CHANGELOG.md

```diff
@@ -64,6 +64,7 @@
 * Fixed an issue where `RouteController` or `PassiveLocationManager` sometimes snapped the user’s location assuming a path that violated a turn restriction. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
 * Improved performance and decreased memory usage when downloading routing tiles. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
 * Renamed `PassiveLocationManager.startUpdatingLocation(completionHandler:)` to `PassiveLocationProvider.startUpdatingLocation()`. This method now runs synchronously like `CLLocationManager.startUpdatingLocation()`. ([#2823](https://github.com/mapbox/mapbox-navigation-ios/pull/2823))
+* Remove `PassiveLocationManager.startUpdatingLocation` in favour of `PassiveLocationManager.startTripSession()` which works better with new pricing option. ([#3362](https://github.com/mapbox/mapbox-navigation-ios/pull/3362)).
```

Comment:
We should remove the previous point as this one rewrites it from the perspective of 1.x.
Also please mention the addition of `stopTripSession`.

Reply:
Can't remove it completely though I should merge two entries.